scrapy/pipelines/files.py

```diff
@@ -85,6 +85,8 @@ class S3FilesStore(object):
 
     POLICY = 'private'  # Overriden from settings.FILES_STORE_S3_ACL in
                         # FilesPipeline.from_settings.
+    AWS_SECURE_CONNECTION = False
```

Comment:
I think you can drop the `AWS_` prefix here, as it's more about S3/boto than AWS